src/components/MenuPanel/menupanel.css

```diff
@@ -0,0 +1,16 @@
+.MenuPanel {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin-left: 50px;
+    margin-right: 50px;
+}
+
+.MenuPanel > * {
+    /*border: 1px solid red;*/
```

Comment:
Please don't keep the commented code in VCS

Reply:
fixed